picard/tests/test_sklearn.py

```diff
@@ -0,0 +1,92 @@
+# Authors: Pierre Ablin <pierre.ablin@inria.fr>
+#          Alexandre Gramfort <alexandre.gramfort@inria.fr>
+#
+# License: BSD (3-clause)
+import warnings
+import pytest
+
+import numpy as np
+
+from sklearn.utils._testing import assert_array_almost_equal
+from sklearn.utils._testing import assert_warns
+
+from picard import Picard
+
+
+def center_and_norm(x, axis=-1):
+    """ Centers and norms x **in place**
+        Parameters
+        -----------
+        x: ndarray
+            Array with an axis of observations (statistical units) measured on
+            random variables.
+        axis: int, optional
+            Axis along which the mean and variance are calculated.
```

Comment:
bad indent

Reply:
Copy and paste from `sklearn/decomposition/tests/test_fastica.py` ;)